### ISSUE TITLE
Contribution

### ISSUE THREAD

**Issue by jcubic** [braden-w/whispering#88]
Do you accept contribution? According to README, the development is on hold.

**Comment by braden-w** [comment]
Yes! Just updated the [README](https://github.com/braden-w/whispering#contributing). feel free to contribute on docs, more adapters, etc.!!
